examples/modals/persistent.py

```diff
@@ -0,0 +1,59 @@
+import nextcord
+
+TESTING_GUILD_ID = 123456798  # Replace with your testing guild id
+
+class Feedback(nextcord.ui.Modal):
+    def __init__(self):
+        super().__init__(
+            title="Feedback",
+            custom_id="feedback",
+            timeout=None,
```

Comment:
Maybe add a comment saying that to make it persistent, these are required, similar to the views example?